pkg/boot/multiboot/internal/trampoline/trampoline_linux_amd64.s

```diff
@@ -133,7 +133,7 @@ TEXT farjump32(SB),NOSPLIT,$0
 	WORD	$0x18 // code segment
 
 TEXT gdt(SB),NOSPLIT,$0
-	QUAD	$0x0		// 0x0 null entry
```

Comment:
Why are you adding another tab here? My formatter kept them all aligned like it was previously.

Reply:
Tbh I don't know, will remove it.